src/pint/utils.py

```diff
@@ -1580,21 +1620,246 @@ def companion_mass(pb, x, inc=60.0 * u.deg, mpsr=1.4 * u.solMass):
     return x1.to(u.Msun)
 
 
-def ELL1_check(A1, E, TRES, NTOA, outstring=True):
+@u.quantity_input
+def PBDOT(mp: u.Msun, mc: u.Msun, pb: u.d, e: u.dimensionless_unscaled):
```

Comment:
pep8 says functions should be named in lowercase, with words separated by underscores. We can break the rules if there's a good reason, and we don't always follow them in the existing code base regardless, but would a lowercase version be more inconvenient or confusing?

Reply:
That's what I had originally, but I changed these to match the table you made:
https://nanograv-pint.readthedocs.io/en/stable/timingmodels.html
for the PK parameters.  So I don't feel strongly either way, and would be happy to stick with lowercase.

Reply:
I'd rather lowercase. I know the upper-case versions are more standard for the parameters themselves (though I'm not sure how much of that is a PINT convention - I don't think tempo/tempo2 care?) but that might even be a good thing, helping to keep the function names separate from the parameters?